src/risedevtool/src/bin/risedev-dev.rs

```diff
@@ -279,6 +279,18 @@ fn task_main(
                 ctx.pb
                     .set_message(format!("kafka {}:{}", c.address, c.port));
             }
+            ServiceConfig::SchemaRegistry(c) => {
+                let mut ctx =
+                    ExecuteContext::new(&mut logger, manager.new_progress(), status_dir.clone());
+                let mut service = SchemaRegistryService::new(c.clone());
+                service.execute(&mut ctx)?;
+                let mut task =
+                    risedev::TcpReadyCheckTask::new(c.address.clone(), c.port, c.user_managed)?;
```

Comment:
Have you checked whether this is accurate?

Reply:
How can I check?

Reply:
maybe perform a GET request and check the code equals to 200. Here's an example

```
curl -X GET http://localhost:8081/subjects
```